luarocks.yaml

```
@@ -0,0 +1,55 @@
+package:
+  name: luarocks
+  version: 3.11.1
+  epoch: 0
+  description: "Package manager for Lua modules"
+  copyright:
+    - license: MIT
+  options:
+    no-depends: true
+  dependencies:
+    runtime:
+      - curl
+      - gcc
+      - luajit
```

```suggestion
      - git
      - luajit
```

luarocks uses git for some downloads